spec/lib/traject/config/note_fields_spec.rb

```diff
@@ -578,16 +578,31 @@
         MARC::Record.new.tap do |r|
           r.append(
             MARC::DataField.new(
-              '880', '0', '0',
+              '880', indicator1, '0',
               MARC::Subfield.new('6', '520-00'),
               MARC::Subfield.new('a', '本书在综合考察绘画,文人,禅学这三者相互关联的基础上.')
             )
           )
         end
       end
 
-      it 'maps the right fields' do
-        expect(result_field.first[:unmatched_vernacular]).to eq ['本书在综合考察绘画,文人,禅学这三者相互关联的基础上.']
+      context 'has 4 indicator' do
+        let(:indicator1) { '4' }
+
+        it 'maps the right fields' do
+          expect(result_field.length).to be 2
```

Comment:
I think there shouldn't be a Summary unmatched vernacular here?

```suggestion
          expect(result_field.length).to be 1
```

Reply:
No it should be 2. When the indicator is 4 there should be a summary and content advice pair. When there isn't there should just be summary. Did I misunderstand the ticket again?

Reply:
okay I think I understand what you are saying. You are saying 520 should only map to one field. Either Content Advice if the indicator is 4 and Summary if that isn't true?

Reply:
Exactly.